apps/backend/src/libs/modules/plugins/auth/auth.plugin.ts

```diff
@@ -49,9 +45,19 @@ const auth = (app: FastifyInstance, { whiteRoutes }: PluginOptions): void => {
 			}
 
 			request.user = user;
-		} catch {
-			throw new AuthError();
+		};
+
+		if (isWhiteListRoute) {
+			try {
+				await parseToken();
+
+				return;
+			} catch {
+				return;
+			}
 		}
+
+		await parseToken();
```

Comment:
seems like it can be simplified to this, also it's better to rename parseToken to something like authorize user, because this function is not only responsible for parsing token, but also finding and attaching user to request
```suggestion
			try {
				await authorizeUser();
		} catch (error) {
		
		       if (!isWhiteListRoute) {
		            trhow error;
			   }
			}
```